### ISSUE TITLE
Question about normalization

### ISSUE THREAD

**Issue by wade3han** [shelling203/SpecAugment#6]
Hello, I'm wondering how the log-mel spctrograms are normalized to have zero mean value. The paper mentioned about it, and the masking value is 0 because they have zero mean. Am I missing, or not implemented?

**Comment by tongjinle123** [comment]
same question ~ using db scale and using zero normalized scale seems different! which scale do we need?

**Comment by TeaPoly** [comment]
> Hello, I'm wondering how the log-mel spctrograms are normalized to have zero mean value. The paper mentioned about it, and the masking value is 0 because they have zero mean. Am I missing, or not implemented?

Using training data features (here is log-mel spctrograms ) to calculate global mean and standard deviation. And do mean and variance normalization with this coefficient. 
 [CMVN](https://en.wikipedia.org/wiki/Cepstral_mean_and_variance_normalization)
